amt.go

```diff
@@ -309,6 +363,20 @@ func (n *Node) setBit(i uint64) {
 	n.Bmap[0] = n.Bmap[0] | byte(1<<i)
 }
 
+func (n *Node) clearBit(i uint64) {
+	if i > 7 {
+		panic("cant deal with wider arrays yet")
+	}
+
+	if len(n.Bmap) == 0 {
+		panic("invariant violated: called clear bit on empty node")
+	}
+
+	mask := byte(((1 << 8) - 1) - (1 << i))
```

Comment:
`(1 << 8) - 1` is a weird way of typing `0xff`

Reply:
>.>